src/components/__tests__/SidePanel-test.jsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+
+import { options } from '../../globals';
+import { fireEvent, render } from '../../utils/testUtils';
+import SidePanel from '../SidePanel/SidePanel';
+
+describe('SidePanel', () => {
+    it('should have Load present at startup', () => {
+        const screen = render(<SidePanel />);
+        expect(screen.getByText('Load')).toBeDefined();
+    });
+
+    it('should not have LoadSave present at startup', () => {
+        const screen = render(<SidePanel />);
+        expect(screen.queryByText('Save / Export')).toBeNull();
+    });
+
+    it('should have LoadSave present if options.index is not zero', () => {
```

Comment:
Really nice that you added tests. Just a minor comment, perhaps it would be even more useful if this text stated what this means in practice? So `options.index is not zero` basically translates to `the number of samples is non-zero`, or something similar. This would make it easier for someone who is not so intimately familiar with PPK to know why it is behaving like this.

Reply:
Great comment! I will change it before merging